src/index.css

```diff
@@ -1,13 +1,120 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  height: 100vh;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  flex: 1;
+  background: #ccc;
+}
+
+.container-wrapper {
+  max-width: 80rem;
+  margin: 0 auto;
+  padding: 2rem 3rem
+}
+
+.header {
+  background: #504118;
+}
+
+.header a{
+  color: #fff;
+  text-decoration: none;
+
+}
+
+.header a:hover {
+  text-decoration: underline;
+}
+
+.footer {
+  background: #f2cdb0;
+  text-align: right;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100%;
+}
+
+.task-form {
+  width: 50%;
+  margin: auto;
+  background: #aaa;
+  padding: 2rem 3rem;
+  border-radius: 10px;
+}
+
+.task-form fieldset {
+  border: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+}
+
+.task-form legend {
+  font-size: 2rem;
+  font-weight: bold;
+  margin-bottom: 1.5rem
+}
+
+.task-box {
+  font-size: 1rem;
+  padding: .2rem .5rem;
+  border: none;
+  border-radius: 5px;
+}
+
+.task-list {
+  width: 50%;
+  margin: 3rem auto;
+}
+
+.task-list li {
+  cursor: pointer;
 }
+
+.task-list .done {
+  text-decoration: line-through;
+}
+
+.button {
+  font-size: 1rem;
+  padding: .5rem;
+  background-color: darkred;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.button:disabled {
+  opacity: .5;
+  cursor: default;
+}
+
+.tachado{
```

Comment:
Be consistent. Do all the naming in english. Besides, `tachado` is confusing while it has a `'text-decoration': none`